content/ember-4-1-released.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Ember 4.1 Released
+authors:
+  - jared-galanis
+date: 2022-01-03T00:00:00.000Z
+tags:
+  - releases
+  - '2022'
+  - Version-4-1
```

Comment:
Needs to be like this, I think:

```suggestion
  - version-4-x
```

Reference: https://github.com/ember-learn/ember-blog/blob/main/content/ember-3-28-released.md